### DISCUSSION TITLE
Docker unhealthy container

### DISCUSSION THREAD

**Discussion by ghost** [haugene/docker-transmission-openvpn#2754] [Q&A]
I've noticed that my docker container has a status of unhealthy recently and I'm not sure why. Is there a healthcheck that its failing?

**Comment by pkishino** [comment]
yes, if you check the logs it should show something

**Comment by ilike2burnthing** [comment]
Try https://github.com/haugene/vpn-configs-contrib/issues/258#issuecomment-1834541037
